<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomDistributionDialog</class>
 <widget class="QDialog" name="RandomDistributionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Values</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="RandomDistributionGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="labelVerticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Distribution</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="parameter4Label">
            <property name="text">
             <string>p(x) = </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="parameter1Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="parameter2Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="parameter3Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="valueVerticalLayout">
          <item>
           <widget class="QComboBox" name="distComboBox"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="parameter1LineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="parameter2LineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="parameter3LineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="randomDistributionButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>randomDistributionButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>RandomDistributionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomDistributionButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>RandomDistributionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
